lib/py/cli.py

```diff
@@ -11,14 +11,18 @@ def inf(opts):
     if opts['in']:
         return open(opts['in'], 'rb')
     else:
-        return sys.stdin
+        if sys.stdin.isatty():
```

Comment:
pls add a comment that explains why we need this check

Reply:
Added a comment